db.go

```diff
@@ -24,11 +26,83 @@ const (
 var errTimeout = fmt.Errorf("could not connect to database: timed out")
 
 // DB represents a wrapper for SQL DB providing extra methods.
+// DB satisfies the sql.DB interface.
 type DB struct {
 	*sql.DB
+	hist prometheus.Histogram
 }
 
-// Check will attempt to ping the database to see if the connection is still alive.
```

Comment:
Looks like you accidentally removed comment/docs for Check function, which broke the build.

Reply:
If you fix this, I can get this merged. This is really useful.